iothub/device/src/RetryPolicies/IotHubClientNoRetry.cs

```diff
@@ -13,6 +13,16 @@ public class IotHubClientNoRetry : IIotHubClientRetryPolicy
         /// <summary>
         /// Create an instance of this class.
         /// </summary>
+        /// <example>
+        /// <code language="csharp">
+        /// var noRetry = new IotHubClientNoRetry();
+        /// 
+        /// var clientOptions = new IotHubClientOptions(new IotHubClientMqttSettings(IotHubClientTransportProtocol.WebSocket))
+        /// {
+        ///     RetryPolicy = noRetry,
+        /// };
```

Comment:
Consider inlining it as it is pretty simple.

```suggestion
        /// var clientOptions = new IotHubClientOptions(new IotHubClientMqttSettings(IotHubClientTransportProtocol.WebSocket))
        /// {
        ///     RetryPolicy = new IotHubClientNoRetry(),
        /// };
```